pkg/nuclide/ocaml-base/lib/MerlinProcess.js

```diff
@@ -269,6 +270,18 @@ function getPathToMerlin(): string {
     && global.atom.config.get('nuclide.nuclide-ocaml.pathToMerlin') || 'ocamlmerlin';
 }
 
+/**
+ * @return The set of arguments to pass to ocamlmerlin.
+ */
+function getMerlinFlags(): Array<String> {
+  const configVal = global.atom
+    && global.atom.config.get('nuclide.nuclide-ocaml.merlinFlags');
```

Comment:
This won't work at all remotely. I'm fine with adding this, so long as you don't write code in the future that depends on certain flags having been passed.